packages/auth/src/index.ts

```diff
@@ -111,20 +111,28 @@ export function initIocContainer(
     })
   })
 
-  container.singleton('openApi', async () => {
-    const authServerSpec = await createOpenAPI(
-      path.resolve(__dirname, './openapi/auth-server.yaml')
-    )
-    const resourceServerSpec = await createOpenAPI(
-      path.resolve(__dirname, './openapi/resource-server.yaml')
-    )
-    const idpSpec = await createOpenAPI(
-      path.resolve(__dirname, './openapi/id-provider.yaml')
-    )
-    return {
-      authServerSpec,
-      resourceServerSpec,
-      idpSpec
+  container.singleton('openApi', async (deps: IocContract<AppServices>) => {
+    try {
+      const authServerSpec = await createOpenAPI(
```

Comment:
what do you think of `openPaymentsAuthServerSpec`? Likewise with `openPaymentsResourceServerSpec` in the `backend`?

Trying to see if there is a better way of distinguishing the several different spec we have

Reply:
This one here is actually `authServerSpec` (from https://github.com/interledger/rafiki/blob/main/packages/auth/src/openapi/resource-server.yaml`)
I think `openPaymentsAuthServerSpec` would be https://github.com/interledger/open-payments/blob/main/openapi/schemas.yaml

What if we just called the first one introspection/tokenIntrospection for both the file name (make it consistent in both `auth` and `backend`) and the variable name?

Reply:
I'm gonna change https://github.com/interledger/rafiki/blob/main/packages/auth/src/openapi/resource-server.yaml to `token-introspection.yaml` with
```suggestion
      const tokenIntrospectionSpec = await createOpenAPI(
```
and not
```suggestion
      const tokenIntroSPECtion = await createOpenAPI(
```
and keep the other file and variable names as is.

Reply:
The `tokenIntrospectionSpec` change makes sense, but it should be replacing`resourceServerSpec` variable (the one that contains the `introspection` route), right?

Reply:
Ha, yeah. I was thinking these comments were for `packages/backend/src/index.ts` :upside_down_face: